### ISSUE TITLE
Links still points to geeksforgeeks in html and do not work in pdf

### ISSUE THREAD

**Issue by sahilshekhawat** [CuriousLearner/GeeksForGeeksScrapper#3]

**Comment by CuriousLearner** [comment]
Yeah, links are not made to work locally right now. It's not a completely off-line version. If you want to work on it, please let me know here and issue a PR. I will gladly merge it.

Also, would like to know which links aren't working in PDF?

**Comment by sahilshekhawat** [comment]
Please refer to https://github.com/CuriousLearner/GeeksForGeeksScrapper/pull/4

PDF don't have any links.

**Comment by CuriousLearner** [comment]
Okay but since we're not storing categories/tags so these links won't work offline right?

So, how would this be useful. Also, links are working for PDFs.

**Comment by CuriousLearner** [comment]
Hello @sahilshekhawat, waiting for you reply on this :)
